### PR TITLE
fixed input field size

### PR DESCRIPTION
**Pull request by elite16op**
#232 

Closes #

### What new changes did you make? Tick all applicable boxes
- [ ] Added new abbreviation
- [✔️] Fixed something in the source code
- [ ] Added a new feature
- [ ] Fixed the docs (README.md, CONTRIBTUING.md, etc)

### Describe the new changes you added.
I have increased the width of input field size

### Share a screenshot of new changes

Screenshot (496)

### CODE REVIEW

**Comment by elite16op** [comment]
The issue of the field input size has been fixed

**Review by rupali-codes — COMMENTED** [review]
great, looks awesome!

File: src/components/form.jsx [hunk 1]
```diff
@@ -103,7 +103,7 @@ const Form = () => {
               <input
                 type="text"
                 placeholder="Search slang full meaning..."
-                className="flex-1 w-1/2 h-11 rounded-full ml-2 border-none outline-none placeholder:text-gray bg-ash"
+                className="flex-1 w-[14rem] h-11 rounded-full ml-2 border-none outline-none placeholder:text-gray bg-ash"
```
Comment (Njong392) [inline src/components/form.jsx:106]: This looks great! Thank you very much 🎉

**Review by Njong392 — CHANGES_REQUESTED** [review]
Thank you for the changes, but I see that you have made changes to the `package-lock.json` file. To revert this, please check out [this guide](https://gist.github.com/kumarsonsoff3/fbc6ecde519250a9f0a4d7ccc9993a1a).

File: src/components/form.jsx (hunk 1 above)
Comment (elite16op) [reply src/components/form.jsx:106]: I have reverted the changes of package.json file. Kindly check it.

**Review by Njong392 — APPROVED** [review]
Looks good to me now! Thank you very much for contributing 🎉

**Pull request merged**